diff --git a/setup.py b/setup.py
index 6bd490b..4be9f67 100644
--- a/setup.py
+++ b/setup.py
@@ -56,7 +56,7 @@ setup(
 
     # You can just specify the packages manually here if your project is
     # simple. Or you can use find_packages().
-    packages=["otfmi"],
+    packages=["otfmi", "otfmi.example"],
 
     # List run-time dependencies here.  These will be installed by pip when
     # your project is installed. For an analysis of "install_requires" vs pip's
